demo/angular/src/app/app.component.ts

```diff
@@ -21,18 +21,22 @@ export class AppComponent {
   }
 
   initializeApp() {
-    this.platform.ready().then(() => {
+    this.platform.ready().then(async () => {
```

Comment:
I would not combine `then` with `async` 

Why not this?
```
async initializeApp() {
     await this.platform.ready();

     this.statusBar.styleDefault();
     this.splashScreen.hide();

      /**
       * initialize() require after platform.ready();
       */
      await AdMob.initialize({
        requestTrackingAuthorization: true,
        testingDevices: ['2077ef9a63d2b398840261c8221a0c9b'],
        initializeForTesting: true,
      });

      AdMob.globalSettings({
        volume: 0.1,
      })

}

```

BTW. why should `AdMob.initialize()` be called after platform ready? I think that was needed for cordova but afaik capacitor registers all plugins in the bridge before the view is there. Or am I wrong?

Reply:
No, platform ready is not need. I will remove.